### ISSUE TITLE
Convert files to other formats

### ISSUE THREAD

**Issue by 0PandaDEV** [0PandaDEV/Qopy#17]
### Describe your requested feature

A second button called "Convert" next to the information/metadata #5 which then opens a popover with the ability to convert the current selected file to various formats.

Here is a concept of how this could look:

![image](https://github.com/user-attachments/assets/090700f7-c9e5-479f-a552-79ef2dd04bb9)

**Comment by wada3n** [comment]
Hi @0PandaDEV, 
Thank you for the new feature! I think it would be more intuitive and user-friendly if this functionality were incorporated into the existing **Actions** menu. This would create a cleaner interface and make all actions easily accessible from one central location.

**Comment by 0PandaDEV** [comment]
@wada3n Yeah, I was also thinking about that since there are currently not a lot of functions there.

**Comment by 0PandaDEV** [comment]
Here is a concept how that could look like:

![image](https://github.com/user-attachments/assets/84cc3b97-b935-4ba5-b0c5-78daa896d584)

**Comment by wada3n** [comment]
Perfect, just as it should be.
